src/cluster/setup.rst

```diff
@@ -45,6 +45,28 @@ possible to force an Erlang application to use a specific port rage.
 This documentation will use the range TCP ``9100-9200``. Open up those ports in
 your firewalls and it is time to test it.
 
+Configure And Test The Communication With Erlang
```

Comment:
I suggest using normal "Title Case" capitalization here, to be consistent with the rest of the documentation.

> Configure and Test the Communication with Erlang